doctr/models/detection/differentiable_binarization/base.py

```diff
@@ -80,9 +80,9 @@ def polygon_to_box(
                     max_size = len(p)
             # We ensure that _points can be correctly casted to a ndarray
             _points = [_points[idx]]
-        expanded_points = np.asarray(_points)  # expand polygon
+        expanded_points: np.ndarray = np.asarray(_points)  # expand polygon
         if len(expanded_points) < 1:
-            return None
+            return None  # type: ignore[return-value]
```

Comment:
@frgfm 
Offtopic (has nothing to do with your PR)
But that's a bit weird i have checked the method usage and `None` isn't handled / catched which would end up in an unexpected error :sweat_smile:

Reply:
I checked that because I was a bit worried and there is:
https://github.com/mindee/doctr/blob/main/doctr/models/detection/differentiable_binarization/base.py#L134

Reply:
Got it and if `self.assume_straight_pages` is False than it will fail with _box = None in https://github.com/mindee/doctr/blob/main/doctr/models/detection/differentiable_binarization/base.py#L146